portfolio/src/main/webapp/index.html

```diff
@@ -90,10 +90,31 @@ <h5 class="text-white h4">Menu</h5><br>
     <br><br><br>
   </div>
 
+  <div class="row peach">
+    <div class="col-2">
+    </div>
+    <div class="col-8">
+      <hr>
+    </div>
+  </div>
+
   <div class="row peach">
 	<div class="col center-text">
-      <button class="btn btn-outline-primary" onclick="getHello()">Fetch the hello!</button>
-      <div id="hello-container"></div>
+      <button class="btn btn-outline-primary" onclick="getComments()">Fetch the comments!</button>
+      <div id="comments-container"></div>
```

Comment:
It seems like these are children of the div above. If so, let's indent the tags accordingly.

Reply:
Okay, thanks. This section of code has now been removed, it was not needed.